apps/native/app/src/ui/lib/skeleton/general-card-skeleton.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { useTheme } from 'styled-components/native'
+import { Skeleton } from './skeleton'
+
+export const GeneralCardSkeleton = ({ height }: { height: number }) => {
```

Comment:
Are these used in the widgets? if so then maybe WidgetSkeleton

Reply:
It is also used on the vehicle screen and in the wallet so I will keep as is :)